### DISCUSSION TITLE
How to add partner skills?

### DISCUSSION THREAD

**Discussion by enchanteddaemon** [Lost-MSth/Arcaea-server#107] [Q&A]
Do i edit the server or the game?

**Comment by Lost-MSth** [comment]
If you mean creating a new skill, it is next to impossible. If you mean adding a skill to one character, using the web backend or changing the database can work.

**Reply by enchanteddaemon** [reply]
the first one, do i need to edit the game?

**Reply by Lost-MSth** [reply]
> next to impossible

Except some skills about world mode, I said it is _next to_ impossible to create new skills. However, the tiny possibility reside in your ablities. Supposed that you can obtain the source code of the client somehow, or you can make a plugin to the client via reverse engineering, there will be no difficulty anymore.

**Reply by enchanteddaemon** [reply]
so adding new partner is impossible too?

**Reply by Lost-MSth** [reply]
Yes, it is.
